Sources/Widgets/Widgets3D/ShapeWidget/behavior.js

```diff
@@ -498,11 +498,19 @@ export default function widgetBehavior(publicAPI, model) {
       }
     } else if (model._isDragging) {
       if (model.activeState === model.point1Handle) {
-        model.point1Handle.setOrigin(worldCoords);
-        model.point1 = worldCoords;
+        model.point1 = vtkMath.add(
+          model.point1Handle.getOrigin(),
+          worldDelta,
+          []
```

Comment:
`vtkMath.add(model.point1Handle.getOrigin(), worldDelta, model.point1)` to avoid array instanciation...